apollo-router/src/plugins/authentication/tests.rs

```diff
@@ -36,13 +36,14 @@ fn create_an_url(filename: &str) -> String {
 }
 
 async fn build_a_default_test_harness() -> router::BoxCloneService {
-    build_a_test_harness(None, None, false).await
+    build_a_test_harness(None, None, true, None).await
 }
 
 async fn build_a_test_harness(
     header_name: Option<String>,
     header_value_prefix: Option<String>,
     multiple_jwks: bool,
+    ignore_other_prefixes: Option<bool>,
```

Comment:
I don't think this needs to be an `Option`.
```suggestion
    ignore_other_prefixes: bool,
```
Anywhere you check `None` can be replaced with a check for `false`.